### PR TITLE
Fixed Book Add/Edit UI

### PR DESCRIPTION
**Pull request by AJV009**
- Did some UI edits to book add/edit form
- Added a custom upload button.
Screenshot from 2020-10-08 23-04-46

**Files changed explicitly:** 
`form.scss`
`addbook.php`

### CODE REVIEW

File: public-html/asset/scss/components/form.scss [hunk 1]
```diff
@@ -1,112 +1,126 @@
...
+    padding-right: 115px;
+    padding-left: 10px;
+  }
+  &[type="file"] {
+    display: none;
+  }
+}
+
+.custom-file-upload {
+  border: 1px solid #ccc;
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: use variable

File: public-html/asset/scss/components/form.scss [hunk 2]
```diff
@@ -1,112 +1,126 @@
...
-	height: 40px;
-	width: 30px;
-	padding-top: 10px;
-	padding-right: 10px;
-	box-sizing: border-box;
+  position: absolute;
+  left: 15px;
+  color: rgb(187, 35, 35);
+  font-size: 16px;
+  background: whitesmoke;
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: use variable

File: public-html/asset/scss/components/form.scss [hunk 3]
```diff
@@ -1,112 +1,126 @@
...
+  }
 }
 
 @media screen and (max-width: 600px) {
-	.main input {
-		margin-right: 40px;
-	}
-}
\ No newline at end of file
+  .main input {
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: one selector per line

File: public-html/asset/scss/components/form.scss [hunk 4]
```diff
@@ -1,112 +1,126 @@
...
+    }
+  }
 }
 
 .main span {
-	position: relative;
+  position: relative;
 }
 
 .main i {
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: one selector per line use nested syntax

File: public-html/asset/scss/components/form.scss [hunk 5]
```diff
@@ -1,112 +1,126 @@
...
+  max-width: 380px;
+  margin: 7% auto;
 }
 
 .header {
-	text-align: center;
+  text-align: center;
 }
 
 .header h1 {
```
Comment (rahulsavaria) [inline public-html/asset/scss/components/form.scss]: one selector per line ( check at all the places )

**Review by rahulsavaria — CHANGES_REQUESTED** [review]

File: public-html/asset/scss/components/form.scss (hunk 1 above)
Comment (AJV009) [reply public-html/asset/scss/components/form.scss]: fixed

File: public-html/asset/scss/components/form.scss (hunk 2 above)
Comment (AJV009) [reply public-html/asset/scss/components/form.scss]: fixed

File: public-html/asset/scss/components/form.scss (hunk 3 above)
Comment (AJV009) [reply public-html/asset/scss/components/form.scss]: fixed

File: public-html/asset/scss/components/form.scss (hunk 5 above)
Comment (AJV009) [reply public-html/asset/scss/components/form.scss]: fixed

**Comment by AJV009** [comment]
@rahulsavaria all issues fixed

**Review by rahulsavaria — APPROVED** [review]

**Comment by JayKandari** [comment]
@AJV009 : Please resolve conflicts.

**Comment by AJV009** [comment]
AM DONE WITH IT AM NOT FIXING THIS ANYMORE!

**Pull request closed without merging**